client/cmd/internal/connectivity/connectivity.go

```diff
@@ -115,6 +116,11 @@ func getDefaultDialOptions() []grpc.DialOption {
 			otelgrpc.StreamClientInterceptor(),
 			grpc_prometheus.StreamClientInterceptor,
 		)),
+		grpc.WithKeepaliveParams(keepalive.ClientParameters{
+			Time:                time.Minute,     // send pings every 10 seconds if there is no activity
```

Comment:
are we expecting 1 minute or 10 seconds?

Reply:
1 min, fixing the comment 😢